app/apps/plugins/front_cache/front_cache_helper.rb

```diff
@@ -8,6 +8,7 @@ def front_cache_front_before_load
           current_site.set_option('refresh_cache', false)
         end
 
+        # avoid cache if current visitor is logged in or development
```

Comment:
This change is unrelated to the PR, but if it's being touched anyway can the middle clauses be
```ruby
Rails.env.development? || Rails.env.test?
```